spec/models/export/azure_upload_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Export::AzureUpload do
+  let(:files_map) { { 'file1.csv' => file_1, 'file2.json' => file_2 } }
+  let(:file_1) { File.open(Rails.root.join('spec', 'fixtures', 'users.csv')) }
+  let(:file_2) { File.open(Rails.root.join('spec', 'fixtures', 'bank_holidays.json')) }
+
+  subject(:azure_upload) { Export::AzureUpload.new(files_map) }
+
+  let(:blob_client) { spy('blob_client') }
+
+  around do |example|
+    ClimateControl.modify AWS_S3_EXPORT_BUCKET: 'test-bucket' do
```

Comment:
Should this be `AZURE_CONTAINER_NAME` ..?

Reply:
Doh! Yes, it should. Which has made me realise I'm not actually dynamically setting the container name 🤦‍♂️